### DISCUSSION TITLE
Ability to wait in macro

### DISCUSSION THREAD

**Discussion by laurent-n** [kingToolbox/WindTerm#595] [Ideas]
Hello,

First of all, thanks for your wonderfull job.
I use macro to automatise some frequent opération, sometime I have to wait a little bit the previous command finish before typing again, it would be great to be able to have a kind of sleep in the macro.

Thanks

**Comment by kingToolbox** [comment]
Sorry for the late reply.

Currently `WindTerm` does not support macros, this feature will be implemented in subsequent versions. You can set the sending interval between lines in the [Sender](https://kingtoolbox.github.io/2020/06/20/sender-send-text/) pane, but I don't know if this can meet your needs.

**Comment by laurent-n** [comment]
I use quick command to do that, and it seems the sending interval of the sender pane is not applied here.
I'll wait the macro are finalized.
Thanks again for your great tool.
